__tests__/app.test.js

```diff
@@ -119,4 +119,59 @@ describe('GET/api/reviews', () => {
           expect(res.body.msg).toEqual('Thers is no review id: 0');
         });
     });
+  });
+
+  describe('PATCH api/review/:review_id', () => {
+    test('201 - responds with an object of updated review when pass valid update object', () => {
+      const update = {inc_votes: 5}
+      return request(app)
+        .patch('/api/reviews/2')
+        .send(update)
+        .expect(201)
+        .then((res) => {
+          expect(res.body.review.votes).toEqual(10)
+          expect(res.body.review).toMatchObject({
+            created_at: expect.any(String),
+            category: expect.any(String),
```

Comment:
If we want to expect the object to look a certain way here in our tests - we can expect the exact values for the given review. E.g. we don't expect the category for review 2 to be ANY string, we know that review 2's category is 'dexterity'. The main thing to be checking here though is that the votes for the specified review have increased which you have done 👍